### ISSUE TITLE
/list command non entirely compatible 

### ISSUE THREAD

**Issue by JoseJohnen** [MCMrARM/revolution-irc#341]
So far the list command shows all the info in the server screen, which makes the list to contract to one or two letters specially if you are on a big server (like libera for instance) but this is not only an issue with big servers, the contraction than happend is so big than even for middle servers will hide all but one or two last groups of channels in alphabetical order.

I think the command should open a special window with the content showing, maybe with auto-join when clicking in the name of the channel.

I ask this because is kinda standard in other IRC clients and this is otherwise a perfect android Foss client for it.

**Comment by MCMrARM** [comment]
There is a channel list feature that you can use instead I think?

**Comment by JoseJohnen** [comment]
Where? because i can't find it, if exist i don't see it, and if i use the command "/list" it doesn't really show the data in his own window, only show it in the server command window.

I don't know if this is relevant, but i'm having my version from F-Droid.

**Comment by MCMrARM** [comment]
From the [...] menu (in top right) -> Channel list.

**Comment by JoseJohnen** [comment]
In the top-right menu i only see "Join channel, Message user, Ignore list, Disconnect, Disconnect and close"

**Comment by MCMrARM** [comment]
Oh sorry, it's [...] -> Join channel -> Channel list

**Comment by JoseJohnen** [comment]
You are right! thank you! i never notice it before and i use it from some years up to this point.

May i suggest move it up one level in order to make it more visible?

And thanks again
